compact/far_backward/lumi/spec_homo_cal.xml

```diff
@@ -18,7 +18,7 @@
   </detectors>
 
   <readouts>
-    <readout name="LumiSpecCALHits">
+    <readout name="EcalLumiSpecHits">
```

Comment:
Does it make sense to keep `Lumi` up front here for lexical sorting among the other lumi collections?

Reply:
That was my first line of thinking but then I saw a different "convention" from other CAL readouts.  They seemed to first start with the type: Ecal or Hcal, and then the name of the detector.

Reply:
https://github.com/eic/epic/blob/9eb69289d9f781b10f45109937fa2e3fca797b6b/compact/ecal/backward_PbWO4.xml#L81

Reply:
Yeah, but we've sort of avoided that convention for far forward and far backward... There the overall system is more pertinent.

Reply:
Ok, changed the readout naming scheme back to the way I had it before.